runtime/envelope/envelope.go

```diff
@@ -22,15 +22,22 @@ import (
 	"errors"
 	"fmt"
 	"io"
+	"log/slog"
 	"os"
 	"strconv"
 	"sync"
 
```

Comment:
What do you think about adding the following line right here? 

```go
_ "github.com/ServiceWeaver/weaver"
```

I believe this way, deployers don't have to import `"github.com/ServiceWeaver/weaver"` themselves.

Reply:
Good idea. Done.